domain/domain.go

```diff
@@ -363,6 +366,38 @@ func NewDomain(store kv.Storage, lease time.Duration) (d *Domain, err error) {
 	return d, nil
 }
 
+// LoadPrivilegeLoop create a goroutine loads privilege tables in a loop, it should be called only once
+// in BootstrapSession
+func (do *Domain) LoadPrivilegeLoop(ctx context.Context) error {
+	do.privHandle = &privileges.Handle{}
+	err := do.privHandle.Update(ctx)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	go func(do *Domain) {
+		ticker := time.NewTicker(5 * time.Minute)
```

Comment:
This is a temporarily solution,  I'll optimize privilege table loading strategy later, it would be change then.